builtin/game/falling.lua

```diff
@@ -105,6 +105,19 @@ core.register_entity(":__builtin:falling_node", {
 				textures = textures,
 				glow = def.light_source,
 			})
+		elseif def.drawtype == "mesh" then
+			local itemstring = node.name
+			if core.is_colored_paramtype(def.paramtype2) then
+				itemstring = core.itemstring_with_palette(itemstring, node.param2)
+			end
+			local s = (def.visual_scale or 1) * SCALE * 0.5
```

Comment:
The only difference here to `~= "airlike"` seems to be this line. To reduce code duplication I'd recommend to add the check below and assign `vsize` there accordingly.